hyungjoonYoon/db/migrations/20221122152112_users.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT,
+		name VARCHAR(50) NOT NULL,
+		email VARCHAR(200) NOT NULL,
+		profile_image VARCHAR(1000) NULL,
+		password VARCHAR(200) NOT NULL,
+		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+		PRIMARY KEY(id)
+);
```

Comment:
회원가입 시 유저 정보 중 중복되면 안되는 데이터를 저장해야하는 경우가 있습니다.

따라서 unique contraint를 추가해주셔야 합니다.

```suggestion
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT,
		name VARCHAR(50) NOT NULL,
		email VARCHAR(200) NOT NULL,
		profile_image VARCHAR(1000) NULL,
		password VARCHAR(200) NOT NULL,
		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
		updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
		CONSTRAINT users_email_ukey UNIQUE (email)
		PRIMARY KEY(id)
);
```